### DISCUSSION TITLE
Curious why this creates 2GB .ckpt files

### DISCUSSION THREAD

**Discussion by EricBintner** [TheLastBen/fast-stable-diffusion#138] [Q&A] [answered]
I'm curious what would cause this to create a 2GB .ckpt file when the HuggingFace v1-4 file is 4GB? Is this expected?

**Comment by TheLastBen** [comment, marked as answer]
The 2GB is in half precision which is what is widely used, the quality is exactly the same and a faster loading

**Comment by hashnag** [comment]
For the record for anyone that's curious: it's not just this implementation, the JoePenna Dreambooth also produces 2GB checkpoints (and probably most others).

**Comment by SAC020** [comment]
Trained a model on standard google colab GPU, it maxed out GPU memory but run without crashing.

However, the resulting model is just 2GB, and the quality of generated images is poor. 

When I train on "premium" GPU, I get 4GB files and the quality is better.

Is there a correlation?

Thx

**Reply by TheLastBen** [reply]
In the cell 'start dreambooth', click twice and change `fp16 = True` to `fp16 = False` and see if there is any difference, if so, I'll add back the checkbox

**Reply by SAC020** [reply]
I have obtained much better results, but I've also increased the number of steps, so maybe not a like-for-like experiment. Regardless, I think it's worthwhile adding back the checkbox

**Reply by giteeeeee** [reply]
> increased the number of steps, so maybe not a like-for-like experiment. Regardless, I think it's worthwhile adding

maybe try using the exact same number of steps just to compare in parallel? I'm really curious
